packages/selinux-policy/rules.cil

```diff
@@ -246,6 +246,7 @@
 (allow clock_s measure_t (files (mutate)))
 (allow network_s lease_t (files (mutate)))
 (allow runtime_s cache_t (files (mutate)))
+(allow trusted_s cache_t (files (mutate)))
```

Comment:
You can change the line above from `runtime_s` to `trusted_s` (which includes `runtime_s`).
```